campaigns.py

```diff
@@ -50,11 +50,32 @@ def template(app):
     env['template_contents'] = template.render(env.context)
 
 
+def clear_static_generated_templates():
+    """
+    Our various frontend applications use the wrapper.html.erb and
+    header_footer_only.html.erb templates. They get these templates
+    using the Slimmer gem. And this gem fetches these templates from
+    the static application, located using the ASSET_ROOT.
+
+    When static is deployed there are no generated wrapper.html.erb or
+    header_footer_only.html.erb templates. At the first request of
+    either of these, the application will generate the template. The
+    template will be placed in the public/template directory. From
+    that point on, the templates are served by nginx.
+
+    This function clears the public/template directory to force it to
+    be regenerated to include the emergency campaign.
+    """
+    for template in ('wrapper.html.erb', 'header_footer_only.html.erb'):
+        sudo('rm -r /var/apps/static/public/templates/{}'.format(template))
```

Comment:
I wonder if this should be a Rake task in the static application, so that our Fabric scripts don't need internal knowledge of how the Static application works.